scripts/scenarios/cloud6/qa/no-ssl/qa-scenario-1a-ha-kvm-swift.yaml

```diff
@@ -40,8 +40,9 @@ proposals:
     ha:
       storage:
         shared:
-          device: 10.162.66.1:/srv/nfs/cloud4-ha-database
+          device: ##shared_nfs_for_database##
```

Comment:
currently by the scenario job, to be cleaned up (see card about unifying jobs with JJB in team green)